File: src/main/java/me/oddlyoko/odoo/modules/reference/OdooFileReference.java
package me.oddlyoko.odoo.modules.reference;

import com.intellij.psi.PsiElement;
import com.intellij.psi.impl.source.resolve.reference.impl.providers.FileReferenceSet;
import org.jetbrains.annotations.NotNull;

public class OdooFileReference extends FileReferenceSet {
    public OdooFileReference(@NotNull PsiElement element) {
        super(element);
    }
}

File: src/main/kotlin/me/oddlyoko/odoo/modules/reference/OdooFileReference.kt
package me.oddlyoko.odoo.modules.reference

import com.intellij.psi.PsiElement
import com.intellij.psi.impl.source.resolve.reference.impl.providers.FileReferenceSet

class OdooFileReference(psiElement: PsiElement): FileReferenceSet(psiElement)
